pkg/config/config_linux.go

```diff
@@ -142,11 +154,54 @@ func GenericLsmObjs() (string, string) {
 	return "bpf_generic_lsm_core.o", "bpf_generic_lsm_output.o"
 }
 
+func CGroupMkdirObj() string {
```

Comment:
Not for this PR but we should consider creating a generic `SelectBpfObj()` interface here